pages/your_tasks/dm_coordination.md

```diff
@@ -51,7 +51,8 @@ Once the project is awarded, the data management plan needs to be executed throu
 * Some data management challenges may not have been foreseen at the pre-award stage.
 * Possible difficulties during the execution of the DMP by individual partners may arise. Not all partners have the same skills and resources.
 * Possible problems with data exchange between partners can resurface during the project. 
-* Consider contacting and establishing collaborations for depositing data with key repositories, e.g. setting up an {% tool "ena-compare-data-hubs" %} for depositing sequence data at the {% tool "european-nucleotide-archive" %}.
+* Consider contacting and establishing collaborations for depositing data with key repositories, e.g. setting up an {% tool "ena-compare-data-hubs" %} for depositing sequence data at the {% tool "european-nucleotide-archive" %}
+or collaborating with {% tool "biostudies" %}, that can provide a project-specific data collection for early data sharing and sustainable data publishing (as in, e.g., [this use case](https://www.ebi.ac.uk/biostudies/EU-ToxRisk/studies)).
```

Comment:
I'd suggest to replace "e.g. this use case" with "for example, the case study on (and mention briefly what the use case is)" as it gives reader some more information.